src/main/java/net/syscon/elite/api/resource/impl/SchedulesResourceImpl.java

```diff
@@ -76,10 +76,15 @@ public SchedulesResourceImpl(final SchedulesService schedulesService, final Appo
     }
 
     @Override
-    public List<PrisonerSchedule> getActivities(final String agencyId, final List<String> body, final LocalDate date, final TimeSlot timeSlot, final boolean includeExcluded) {
+    public List<PrisonerSchedule> getActivitiesForBookings(final String agencyId, final List<String> body, final LocalDate date, final TimeSlot timeSlot, final boolean includeExcluded) {
         return schedulesService.getActivities(agencyId, body, date, timeSlot, includeExcluded);
     }
 
+    @Override
+    public List<PrisonerSchedule> getActivities(final List<Long> eventIds) {
+        return schedulesService.getActivities(eventIds);
```

Comment:
can't remember how does security work for this - will anyone with a token be able to access everything?

Reply:
Good catch! Not sure the schedule is too sensitive but will verify the agency like the other methods in the service. At least then it's consistent.